sdks/RepoToolset/tools/Sign.proj

```diff
@@ -19,6 +25,8 @@
     </Exec>
 
     <ItemGroup>
+      <SignToolArgs Condition="'$(OrchestrationMode)' == 'true'" Include='-orchestrationmode' />
```

Comment:
Aren't there two cases:
1) The tool reads -config file and writes -orchestrationconfigfile
2) The tool reads -config file and signs

The distinction between these is the presence of -orchestrationconfigfile


---
In reply to: [152713885](https://github.com/dotnet/roslyn-tools/pull/100#discussion_r152713885) [](ancestors = 152713885)

Reply:
It would be better to rename ```-orchestrationconfigfile```
to something that indicates it's the output. Perhaps ```-outputConfig```? The fact that this is don't for orchestration isn't significant. If ```-outputConfig``` is present we are not signing, just producing another config file.

---
In reply to: [152713975](https://github.com/dotnet/roslyn-tools/pull/100#discussion_r152713975) [](ancestors = 152713975,152713885)